src/main/java/se/sundsvall/alkt/api/model/Owner.java

```diff
@@ -0,0 +1,131 @@
+package se.sundsvall.alkt.api.model;
+
+import java.time.LocalDateTime;
+import java.util.List;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+
+import io.swagger.v3.oas.annotations.media.ArraySchema;
+import io.swagger.v3.oas.annotations.media.Schema;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@Getter
+@Setter
+@Builder(setterPrefix = "with")
+@NoArgsConstructor
+@AllArgsConstructor
+public class Owner {
+
+	@Schema(description = "PartyId (e.g. a personId or an organizationId)", example = "c4794977-a073-484d-adef-564b240db8f8")
+	private String partyId;
+
+	@Schema(description = "Name of the organization", example = "Sundsvalls kommun")
```

Comment:
Is this the applying organization? If so: should we have something else than "Sundsvalls kommun" here?